src/components/settings-menu/DevNotificationsSection.tsx

```diff
@@ -48,13 +48,13 @@ const DevNotificationsSection = () => {
   const { wallets } = useWallets();
   const { chainId } = useAccountSettings();
   const [loading, setLoading] = useState<boolean>(true);
-  const walletIDs = Object.keys(wallets);
+  const walletIDs = Object.keys(wallets!);
```

Comment:
`wallets` is probably always defined, but I'd rather have strong guarantee coming from `useWallets` hook rather than this assumption here.